<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QDialog" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QTreeWidget" name="wifi_treeWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>1</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">网络列表</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnRefreshWIFI">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>17</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新网络</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnWlanOpenControl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>71</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton:hover{background-color:rgba(229, 241, 251,255);}
QToolButton:pressed{background-color:rgb(177, 209, 252);border-style: inset;}
QToolButton{background-color: rgba(73, 150, 254, 0);border-radius:4px;}
QToolButton::menu-indicator{image:none;}</string>
   </property>
   <property name="text">
    <string>WLAN OFF</string>
   </property>
   <property name="icon">
    <iconset resource="MainWindow.qrc">
     <normaloff>:/icon/wifiOFF.png</normaloff>:/icon/wifiOFF.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>461</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>当前连接:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ActiveWifiName_Lab">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
